src/protagonist/API/Features/DeliveryChannels/Converters/OldHydraDeliveryChannelsConverter.cs

```diff
@@ -25,31 +25,42 @@ public class OldHydraDeliveryChannelsConverter
         
         foreach (var wcDeliveryChannel in hydraImage.WcDeliveryChannels)
         {
-            var matchedDeliveryChannel = wcDeliveryChannel switch
+            var matchedDeliveryChannels = wcDeliveryChannel switch
             {
-                AssetDeliveryChannels.Image => new DeliveryChannel()
-                {
-                    Channel = AssetDeliveryChannels.Image,
-                    Policy = hydraImage.ImageOptimisationPolicy == ImageUseOriginalPolicy
-                        ? ImageUseOriginalPolicy
-                        : ImageDefaultPolicy
+                AssetDeliveryChannels.Image => new List<DeliveryChannel>(){ 
+                    new() 
+                    {
+                        Channel = AssetDeliveryChannels.Image,
+                        Policy = hydraImage.ImageOptimisationPolicy == ImageUseOriginalPolicy
+                            ? ImageUseOriginalPolicy
+                            : ImageDefaultPolicy 
+                    },
+                    new() 
+                    {
+                        Channel = AssetDeliveryChannels.Thumbnails
```

Comment:
Should this have default policy, rather than just channel? 

Original ACs in #714 have `"thumbs" ImageDeliveryChannel. Always with default policy.` but I'm not sure if there's any practical difference?